eos/utils/kmatrix-impl.hh

```diff
@@ -0,0 +1,213 @@
+/*
+ * Copyright (c) 2019 Danny van Dyk, Stephan Kürten
```

Comment:
Copyright by authors, not by year. Authors listed in alphabetical order of the last name. ```v``` for me!